<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>index</class>
 <widget class="QMainWindow" name="index">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>index</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">#centralWidget{background-image: url(:/new/prefix1/bg.png);}</string>
   </property>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>140</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background:#fe7023;
color:#ffffff;
font: 12pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBook">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>89</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>按书名查询</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioISBN">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>89</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>按ISBN查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background:#fe7023;
color:#ffffff;
font: 10pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lastButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>560</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background:#fe7023;
color:#ffffff;
font: 10pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>上一页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>560</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background:#fe7023;
color:#ffffff;
font: 10pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>下一页</string>
    </property>
   </widget>
   <widget class="QLabel" name="warnTxt">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>521</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(243, 35, 16)</string>
    </property>
    <property name="text">
     <string>输入警告提示</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>591</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>540</y>
      <width>591</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="_10">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>591</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 202, 98);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_11">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;微软雅黑&quot;;
color:rgb(0, 85, 0);
background:rgb(0, 243, 117);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_13">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>245</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_14">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_12">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 0);
font: 10pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_15">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>220</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background:#fe7023;
color:#ffffff;
font: 12pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_21">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;微软雅黑&quot;;
color:rgb(0, 85, 0);
background:rgb(0, 243, 117);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_24">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_25">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>300</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background:#fe7023;
color:#ffffff;
font: 12pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_20">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>591</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 202, 98);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_23">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>325</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_22">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 0);
font: 10pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_31">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;微软雅黑&quot;;
color:rgb(0, 85, 0);
background:rgb(0, 243, 117);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_34">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>420</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_35">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>380</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background:#fe7023;
color:#ffffff;
font: 12pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_30">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>591</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 202, 98);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_33">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>405</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_32">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 0);
font: 10pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_41">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>460</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;微软雅黑&quot;;
color:rgb(0, 85, 0);
background:rgb(0, 243, 117);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_44">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>500</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_45">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>460</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background:#fe7023;
color:#ffffff;
font: 12pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_40">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>450</y>
      <width>591</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 202, 98);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_43">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>485</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_42">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>460</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 0);
font: 10pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="warnTxt_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>560</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(243, 35, 16)</string>
    </property>
    <property name="text">
     <string>操作警告提示</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
font: 12pt &quot;微软雅黑&quot;;</string>
    </property>
   </widget>
   <zorder>_10</zorder>
   <zorder>txtEdit</zorder>
   <zorder>_40</zorder>
   <zorder>_30</zorder>
   <zorder>_20</zorder>
   <zorder>searchButton</zorder>
   <zorder>radioBook</zorder>
   <zorder>radioISBN</zorder>
   <zorder>loginButton</zorder>
   <zorder>lastButton</zorder>
   <zorder>nextButton</zorder>
   <zorder>warnTxt</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>_11</zorder>
   <zorder>_13</zorder>
   <zorder>_14</zorder>
   <zorder>_12</zorder>
   <zorder>_15</zorder>
   <zorder>_21</zorder>
   <zorder>_24</zorder>
   <zorder>_25</zorder>
   <zorder>_23</zorder>
   <zorder>_22</zorder>
   <zorder>_31</zorder>
   <zorder>_34</zorder>
   <zorder>_35</zorder>
   <zorder>_33</zorder>
   <zorder>_32</zorder>
   <zorder>_41</zorder>
   <zorder>_44</zorder>
   <zorder>_45</zorder>
   <zorder>_43</zorder>
   <zorder>_42</zorder>
   <zorder>warnTxt_2</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
